docs/getting_started.md

```diff
@@ -0,0 +1,21 @@
+# Getting Started
+
+## Installation
+
+You can install `mteb` using [pip](https://pip.pypa.io/en/stable/getting-started/) simply by running:
+
+```bash
+pip install mteb
+```
+
+??? tip "Model Specific Installations"
+
+    If you want to run certain models implemented within mteb you will often need some additional dependencies. These can be installed using:
+
+    ```bash
+    pip install mteb[openai]
+    ```
+
+    If a specific mdel requires a dependency it will raise an error with the recommended installation. To get an overview of the implemented models see [here](missing).
```

Comment:
```suggestion
    If a specific model requires a dependency it will raise an error with the recommended installation. To get an overview of the implemented models see [here](missing).
```